app/decorators/keyword_decorator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class KeywordDecorator
```

Comment:
This decorator we are using in the view, but based on our [convention](https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#decorators), decorators should be used only for the backend, for the view, it should be the presenter. So I think this class should be a [presenter class](https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#presenters)

![Screen Shot 2021-12-03 at 12 56 35 PM](https://user-images.githubusercontent.com/6076762/144553073-c851d185-fc3a-4db4-81ef-4a267f859c2c.png)

Reply:
Thanks for the detailed comment 🙌. I fixed in [`f42675d` (#40)](https://github.com/suho/google-search-ruby/pull/40/commits/f42675dd1e9b7fdfe0b06e96e5e8709d750dd695)